scripts/pdf/high-contrast-dark.css

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: MIT */
```

Comment:
Optional fix: separate PR into 2 ones based on directories changed: everything in `scripts/pdf` can be put in a separate PR.